docs/site/Tutorials.md

```diff
@@ -0,0 +1,41 @@
+---
+lang: en
+title: Tutorials
+keywords: LoopBack 4.0
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Tutorials.html
+---
+
+LoopBack 4 comes with the following tutorials:
+
+- **[hello-world](https://github.com/strongloop/loopback-next/tree/master/examples/hello-world)**:
+  Tutorial on setting up a simple hello-world application using LoopBack 4.
+
+- **[todo](todo-tutorial.md)**: Tutorial on building a simple application with
+  LoopBack 4 key concepts using bottom-up approach.
+
+- **[todo-list](todo-list-tutorial.md)**: Tutorial on introducing related models
+  and building their API from the Todo tutorial
+
+- **[log-extension](https://github.com/strongloop/loopback-next/tree/master/examples/log-extension)**:
```

Comment:
Should we extract the tutorial part from log-extension's README into a proper tutorial page in our docs?

See https://github.com/strongloop/loopback-next/blob/c268d5a48a7205719c9df0436fd70cd6c6659a1e/examples/log-extension/README.md#tutorial

This is what we have in other tutorial projects, this snippet is cross-posted from `examples/todo/README.md`:

```md
## Tutorial

To follow this tutorial, begin with the
[Create your app scaffolding](http://loopback.io/doc/en/lb4/todo-tutorial-scaffolding.html)
section
```

Reply:
Another point to discuss with @dhmlau @strongloop/sq-lb-apex

Reply:
It would be good to be consistent with the todo tutorial.  However, I'm looking at the README for the `log-extension` from the above link, it looks more like an example to me?  It has some description on the changes, but not really step by step showing how to create an extension. 

Thoughts?

Reply:
Reading `log-extension` again, @bajtos, you're right that it has some instructions in how to build it, thus fits as tutorial.  We can extract the steps to sidebars in this PR or a follow up.  I'm good either way.